### DISCUSSION TITLE
Support collecting kubernetes events from the apiserver

### DISCUSSION THREAD

**Discussion by JeffLuoo** [fluent/fluent-bit#5425] [Ideas]
**Describe the solution you'd like**

This thread is going to brainstorm and collect some ideas of using fluent-bit to collect kubernetes events.

cc:
@agup006 @patrick-stephens 

**Describe alternatives you've considered**

**Additional context**

**Comment by pwhelan** [comment]
Would the idea be to watch the API server for changes in the status of Deployments, Pods and other relevant objects? Would we want to expose the raw data from the Conditions and Status of these?
